reports.py

```diff
@@ -686,27 +705,60 @@ def plot_coverage(
         # Set the tick labels font
         for label in (ax.get_xticklabels() + ax.get_yticklabels()):
             label.set_fontsize(font_size)
-
+            
+        # Binning
+        bin_size = 1
+        if bin_large_plots:
+            inner_plot_width_inches = ax.get_window_extent().transformed(fig.dpi_scale_trans.inverted()).width
+            inner_plot_width_px = inner_plot_width_inches * fig.dpi # width of actual plot (sans whitespace and y axis text)
+            bins_per_pixel = 1 # increase to make smaller (but less visible) bins
+            
+            # Parse summary statistic
+            if(binning_summary_statistic.lower() == "min" or binning_summary_statistic.lower() == "minimum"):
+                binning_summary_statistic = "min" # for y axis label
+                binning_action = min
+            else:
+                binning_summary_statistic = "max"
```

Comment:
Using `binning_action.__name__` where you want a string version of the binning function name would obviate the need to store it separately

Reply:
Removed that whole block of code and replaced with

`binning_action = eval(binning_summary_statistic)`

since `binning_summary_statistic` is constrained by the `choices` now.

Reply:
Using `eval` was my first instinct as well, despite the fact that it always feels a little wrong, even with constraints. It may be nice to preserve the ability to do it by name in case we ever add an inline function for another bin stat (threshold, q score filter, etc.)